packaging/packager_pip.sh

```diff
@@ -12,9 +12,11 @@ readonly release_tag="${RELEASE_TAG:-}"
 
 # disable tests and support of backends based on python versions
 if [[ $python_version == '3.9' ]]; then export DISABLE_AUTOPY=1; fi
-
```

Comment:
Please keep this empty line to separate the upper from the lower block of lines.